packages/server/src/sdk/app/rows/search/sqs.ts

```diff
@@ -95,14 +95,29 @@ function buildTableMap(tables: Table[]) {
     // update the table name, should never query by name for SQLite
     table.originalName = table.name
     table.name = table._id!
+    // need a primary for sorting, lookups etc
+    table.primary = ["_id"]
```

Comment:
Do we not need a check here to see if the table already has a primary key set?

Reply:
This will never happen in for internal tables (SQS) but I can add a throw just incase.